src-graphql/client.js

```diff
@@ -87,4 +87,67 @@ export default class Client {
       return response.model.node;
     });
   }
+
+  fetchQueryProducts(queryObject, query = productConnectionQuery()) {
+    let queryArgString = '';
+    const options = {};
+
+    if (queryObject) {
+      if (queryObject.title) {
+        queryArgString = queryArgString.concat(`title:'${queryObject.title}' `);
+      }
+      if (queryObject.updatedAtMin) {
+        queryArgString = queryArgString.concat(`updated_at:>='${queryObject.updatedAtMin}' `);
+      }
+      if (queryObject.createdAtMin) {
+        queryArgString = queryArgString.concat(`created_at:>='${queryObject.createdAtMin}' `);
+      }
+      if (queryObject.productType) {
+        queryArgString = queryArgString.concat(`product_type:'${queryObject.productType}' `);
+      }
+      if (queryObject.limit) {
```

Comment:
@minasmart should we keep the `limit` name or just use `first` to match up with GraphQL?